CreeDictionary/API/admin.py

```diff
@@ -50,6 +48,8 @@ def queryset(self, request, queryset):
             return queryset.filter(as_is=True)
 
 
+# todo: make admin search "extended"
+# todo: show inflection definition instead of lemma definition
```

Comment:
about the first todo. I searched for "nipa" on the admin console when I was debugging. the result "nipâ" didn't show up until I added the circumflex. Considering it's not worth a full issue and not important, I'll just add a todo here.